kratos/processes/entity_erase_process.cpp

```diff
@@ -0,0 +1,172 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/kratos_flags.h"
+#include "processes/entity_erase_process.h"
+#include "utilities/variable_utils.h"
+#include "utilities/auxiliar_model_part_utilities.h"
+
+namespace Kratos
+{
+template<class TEntity>
+EntitiesEraseProcess<TEntity>::EntitiesEraseProcess(
+    ModelPart& rModelPart,
+    const bool RemoveFromAllLevels,
+    const bool AssignFlag,
+    const bool RemoveBelongings
+    )
```

Comment:
It would be better to pass a Flags instead of 3 bool Consider that with this implementation adding another option in future would change the API.

Reply:
Meanwhile, assign flag was not really descriptive for me. checking the code I reallized that it is eraser all in model part.

Reply:
Ok, I haven't see this

Reply:
I just checked that this is already done, why doesn't appear as outdated?

Reply:
I still cannot see the change of assign flag

Reply:
Looks like a bug of GitHub:
![imagen](https://user-images.githubusercontent.com/19991680/56193262-848c6180-6030-11e9-883a-29bb0ae1d70a.png)

Reply:
I mean the flag itself which is still `ASSIGN_FLAG`. I understood that you get this as if you should set the flag by yourself but the final intention is to erase all entities in the given sub modelpart. Isn't it?